apps/src/aichat/redux/aichatRedux.ts

```diff
@@ -78,64 +89,122 @@ const initialState: AichatState = {
   currentAiCustomizations: EMPTY_AI_CUSTOMIZATIONS,
   savedAiCustomizations: EMPTY_AI_CUSTOMIZATIONS,
   fieldVisibilities: DEFAULT_VISIBILITIES,
+  viewMode: ViewMode.EDIT,
+  hasPublished: false,
 };
 
 // THUNKS
 
 // This thunk saves a student's AI customizations using the Project Manager (ie, to S3 typically),
 // then does a comparison between the previous and current saved customizations in order to
 // output a message to the chat window with the list of customizations that were updated.
-export const updateAiCustomization = createAsyncThunk(
-  'aichat/updateAiCustomization',
+export const saveAiCustomization = createAsyncThunk(
```

Comment:
very minor / optional - name this `updateAiCustomizations` to match "Update" button and keep the shared function just `saveAiCustomizations`?

Reply:
Commented on this already but I think this would be more clear as well.

Reply:
Ok you guys win, updated

Reply:
😁 Thanks for iterating on this!